gsa/src/index.js

```diff
@@ -19,8 +19,24 @@ import React from 'react';
 
 import ReactDOM from 'react-dom';
 
+import * as Sentry from '@sentry/react';
+
+import {isDefined} from 'gmp/utils/identity';
+
 import App from './web/app.js';
 
+const {sentryDSN, sentryEnvironment} = global.config;
+
+const GSA_VERSION = process.env.REACT_APP_VERSION || '21.10';
```

Comment:
Can we move this to a version module and import it from there to not duplicate the version settings. Should be imported in the about page too.